src/env/env-stub.ts

```diff
@@ -0,0 +1,15 @@
+export class EnvStub {
+  private oldEnv: { [key: string]: string | undefined } = {}
+  public constructor(keys: string[]) {
+    for (const key of keys) {
```

Comment:
Would it make sense for `keys` to default to an array of all connection strings?

Reply:
I don't think so as we don't have it easily available here, the list changes and it's not uncommon to use a bunch of other strings such as URLs, credentials, etc.